microsoft-365/security/office-365-security/remediate-malicious-email-delivered-office-365.md

```diff
@@ -37,19 +37,23 @@ Admins can take required action on emails, but to get those actions approved, th
 *Manual hunting* occurs when security teams identify threats manually by using the search and filtering capabilities in Explorer. Manual email remediation can be triggered through any email view (*Malware*, *Phish*, or *All email*) after you identify a set of emails that need to be remediated.
 
 > [!div class="mx-imgBorder"]
-> [![Manual hunting in Office 365 Threat Explorer by date.](../../media/tp-RemediationArticle1.png)](../../media/tp-RemediationArticle1.png#lightbox)
+> ![image](https://user-images.githubusercontent.com/78144677/161120340-282f579a-2287-40f1-9a19-a54011ecd6cd.png)
```

Comment:
Soumya, your image doesn't have good naming conventions (used by Google SEO for ranking) and no Alternative Text, used by screen readers for accessibility. Take a look at the original graphic: [![Manual hunting in Office 365 Threat Explorer by date.](../../media/tp-RemediationArticle1.png)](../../media/tp-RemediationArticle1.png#lightbox). This is the minimum we would need here.